app/src/main/java/com/example/pokedexcompose/MainActivity.kt

```diff
@@ -9,25 +9,66 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Home
 import androidx.compose.material.icons.outlined.Home
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarColors
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
 import androidx.navigation.compose.rememberNavController
 import com.example.pokedexcompose.ui.list.pokemonListRoute
 import com.example.pokedexcompose.ui.navigation.PokemonNavHost
 import com.example.pokedexcompose.ui.theme.PokedexComposeTheme
 import com.pokedexcompose.designsystem.components.bottomnavigation.BottomNavigationBar
 import com.pokedexcompose.designsystem.components.bottomnavigation.tabs.model.BottomNavigationTabModel
 
+@ExperimentalMaterial3Api
 class MainActivity : ComponentActivity() {
     @RequiresApi(Build.VERSION_CODES.P)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             PokedexComposeTheme {
                 val navController = rememberNavController()
+                val initialColor = MaterialTheme.colorScheme.surfaceContainer
+                var title by remember {
+                    mutableStateOf("Pokemon")
```

Comment:
sera que ja vale a gente por no resource?

Reply:
O texto pokemon? Se for sim , deixei pera definirmos qual seria